### ISSUE TITLE
🐛Old releases removed from apt repository

### ISSUE THREAD

**Issue by F21** [cloudflare/cloudflared#914]
**Describe the bug**
Not 100% sure if this is actually a bug, but I am filing this as a bug as it's seems to be the best category.
The apt repository seems to only contain the latest release, meaning that old releases are removed. It makes perfect sense that users should be running the latest version from a security and stability perspective. However, in our case, we are using Packer to build machine images and have versions of installed software such as Cloudflared pinned in our code. The problem is that everytime a new version of Cloudflared is released, the old versions are removed from the apt repository, so if we were to update parts of our image other than the Cloudflared version, the build will fail. A lot of other security-focused software (such as Vault) keeps older versions of in their apt repository around.

**To Reproduce**
1. Add apt repository to your machine.
2. Run `apt show cloudflared`
3. See only one (latest) version.

**Expected behavior**
It would be nice if old releases are kept in the apt repository.

**Environment and versions**
 - OS: Ubuntu
 - Architecture: AMD x64
 - Version: n/a

**Logs and errors**
n/a

**Additional context**
n/a

**Comment by sudarshan-reddy** [comment]
Indeed. And good observation. This is definitely something we should do eventually. I've deferred priority and will take a look at it once my plate is a bit empty.
